### ISSUE TITLE
Debian and Samurai install scripts are not executable.

### ISSUE THREAD

**Issue by karandesai-96** [owtf/owtf#573]
While running **`install/install.py`** script in Debian, this error is encountered :

```
/home/darth_sidious/Documents/owtf/install/debian/install.sh: 53:
/home/darth_sidious/Documents/owtf/install/debian/install.sh:
/home/darth_sidious/Documents/owtf/install/debian/pref.sh: Permission denied
```

On further inspection of scripts under install directory:
![image](https://cloud.githubusercontent.com/assets/10494087/13546090/cbacedfc-e2c8-11e5-8997-6f10dfb2dfe3.png)

It clearly states that install scripts under **debian** and  **samurai** are not marked executable. These files must be marked executable and made available on upstream HEAD, so the developers need not mark them executable manually on cloning the repository, nor do they face merge conflicts as the executable bit of scripts on local machines is _set_ while that of upstream isn't.

**Comment by delta24** [comment]
A possibly clean solution is to add either `sh` in front of the script names everywhere. That way we don't play with permissions and source remains clean.

**Comment by delta24** [comment]
Fixed in 00acc5b9e12c056fb3fd21f02fe434f54eaa823f (wrong issue referenced in merge commit)
